test/resources/layouts/simple-layout.pug

```diff
@@ -0,0 +1,3 @@
+doctype html
```

Comment:
Where is this file used?

Reply:
Its a layout that is referenced in another template, the filename was wrong in the template that references it, fixed now.